### PR TITLE
Script for finding groups inconsistencies

### PR DESCRIPTION
**Pull request by balcirakpeter**
- In perun db was created new script find_groups_inconsistencies,
  which checks wheter there are groups with some relationship where
  subgroup contains a member which does not exists in the parent group as
  Indirect.

### CODE REVIEW

**Comment by balcirakpeter** [comment]
I have realized that the script does not work right when there is the same member in two groups which are subgroups of the same parent. Therefore, I have to fix it.

**Comment by balcirakpeter** [comment]
Fixed, ready for review.

**Review by stavamichal — APPROVED** [review]

File: perun-db/find_group_inconsistencies [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+			}
+			push(@all_sub_groups, $sub_group);
+		}
+	}
+	my $next_indirect_members = prepare_structure_of_indirect_members(\@all_sub_groups);
+	push(@already_processed_groups, keys %$current_indirect_members);
+	check_against_next_group_level($next_indirect_members);
+}
+
+sub prepare_structure_of_indirect_members {
```
Comment (stavamichal) [inline perun-db/find_group_inconsistencies:104]: Please add documentation of this function.

File: perun-db/find_group_inconsistencies [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+		find_inconsistencies_in_vo($selected_vo);
+	}
+}
+disconnect $dbh;
+
+#
+# Find group inconsistencies in specific vo
+#
+sub find_inconsistencies_in_vo {
+	my $vo = shift;
```
[the remark this answers is not in the document]
Comment (balcirakpeter) [reply perun-db/find_group_inconsistencies:66]: Fixed

File: perun-db/find_group_inconsistencies (hunk 1 above)
Comment (balcirakpeter) [reply perun-db/find_group_inconsistencies:108]: Added

**Review by Vojtech-Sassmann — APPROVED** [review]

**Pull request merged**